.github/renos_updated.sh

```diff
@@ -21,5 +23,5 @@ do
    fi
 done
 
-echo Please add or update a release note in ./releasenotes >&2
+echo Please add or update a release note in ./spec_releasenotes or ./ ast_releasenotes >&2
```

Comment:
```suggestion
echo Please add or update a release note in ./spec_releasenotes or ./ast_releasenotes >&2
```

Reply:
Fixed in https://github.com/openqasm/openqasm/pull/519/commits/d6a45d0c94c6d86d61610542b7bb6e09349f9742